task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -445,17 +448,22 @@ def _register_pipe_readers(self, stdout: socket, stderr: socket, requests: socke
         # alternatives are used automatically) -- this is a way of having "event-based" code, but without
         # needing full async, to read and process output from each socket as it is received.
 
-        self.selector.register(stdout, selectors.EVENT_READ, self._create_socket_handler(self.log, "stdout"))
+        target_loggers: tuple[FilteringBoundLogger, ...] = (self.log,)
+        if self.verbose_logs_to_main_logger:
+            target_loggers += (log,)
```

Comment:
it seems odd that `self.log` would not go to stdout but `log` would go to stdout.  so if i use  the global `log`, it will not go to task logs? 

shouldn't it be more idiot proof than that, in the sense that, whichever log you a re using, it would go to both?

if `log` is a special "stdout only" logger, it should probably be named specially to reflect that

Reply:
`self.log` is the process specific logger, which is configured to write to the file.

`log` is the top level module scoped generic logger.

Would renaming `self.log` to `self.process_log`  or `process_logger` help do you think?